AutomatticTracks/src/main/java/com/automattic/android/tracks/crashlogging/performance/internal/SentryPerformanceMonitoringWrapper.kt

```diff
@@ -0,0 +1,11 @@
+package com.automattic.android.tracks.crashlogging.performance.internal
+
+import io.sentry.ITransaction
+import io.sentry.Sentry
+
+class SentryPerformanceMonitoringWrapper {
```

Comment:
This class makes `PerformanceTransactionRepository` testable. We couldn't mock direct `Sentry.startTransaction` otherwise.